scalability_and_performance/cnv-performance-scale-resilience-practices/recommended-node-practices.adoc

```diff
@@ -0,0 +1,80 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="recommended-node-practices"]
+= Recommended node practices
+include::_attributes/common-attributes.adoc[]
+:context: recommended-node-practices
+
+toc::[]
+
+This topic provides recommended node practices for a Container Native Virtualization environment to be performant, scalable and resilient in {product-title}.
+
+If a node fails and Node Health Checks are not deployed with a appropriate remediation strategy - Self Node Remediation (SNR) and Fence Agents Remediation (FAR) on your cluster, Virtual Machines ( VMs ) will encounter downtime until the node recovers as they are not automatically migrated to other available nodes by default.
+
```

Comment:
@dominikholler thanks for the pointer! So the idea is to not only document node fencing mechanism but also tunings to set the expectations in terms of remediation timing for large and dense clusters ( with various VM counts ) and as well as to improve the performance. In addition to it, we are documenting the capacity guidance to make sure to avoid VMs downtime that we experienced during the chaos testing while make sure the nodes are not over or under utilized. We will continue to expand and maintain it for each of the releases. Thoughts?